### DISCUSSION TITLE
Seq on AWS

### DISCUSSION THREAD

**Discussion by simon-chester** [datalust/seq-tickets#1477] [Help] [answered]
Hi, 

I'm just setting-up Seq for a new client. They use AWS, but I'm quite new to it (having mainly lived in the Azure/on-prem world upto now).

Ideally, I'd like to just spin-up a container using the docker-hub image, but I'm confused about how to persist data to EBS.

Can anyone point me to an idiot's guide on setting-up Seq in AWS?

Thanks, Simon.

**Comment by liammclennan** [comment, marked as answer]
Unfortunately, it is not easy to attach EBS to an AWS hosted container. Here is [some documentation](https://docs.aws.amazon.com/AmazonECS/latest/bestpracticesguide/storage-dockervolumes.html). 

The recommended approach in AWS is still to use EC2 and EBS and [install the docker image](https://docs.datalust.co/docs/docker-deployment-overview). 

AWS provide [instructions for configuring Docker on EC2](https://docs.aws.amazon.com/AmazonECS/latest/userguide/docker-basics.html).

**Reply by simon-chester** [reply]
Thanks for getting back to me. 
As you suggested, we went with EC2 + EBS + docker (on a t4/ARM64!) and it's working fine.  Thanks again.

**Reply by liammclennan** [reply]
Excellent. Good choice going with Graviton.
